### DISCUSSION TITLE
"Anonymous Usage Statistics": Opt-Out vs. Opt-In

### DISCUSSION THREAD

**Discussion by eantowne** [GNS3/gns3-gui#3351] [Ideas]
I just noticed that there is an option for opt-out of anonymous usage statistics (telemetry). I honestly did not even realize that telemetry was being collected. Now that I do, I have a couple of thoughts that I would like to start a discussion around, prior to submit issues.

1. Telemetry collection is always a contentious subject, even more so on FOSS. At the same time telemetry can be a lynchpin for FOSS projects to garner sponsors and support.
2. Personally, Opt-In is preferred over Opt-Out
3. There should be something to notify the user of telemetry collection if Opt-Out is the standard. This also needs to be applied for the Web-UI also

Anyone else have any thoughts?

**Comment by grossmj** [comment]
We use Google Analytics by sending some events from the desktop Ui but this is rather limited because GA is more targeted for websites. We don't even use the (limited) statistics at the moment so I am thinking to either remove it completely or opt-in by default.

**Comment by eantowne** [comment]
Ideally, I vote for removing, if there is no actual usage. If you foresee some use, then I would suggest opt-in, or a dialog to enable/disable during Setup Wizard. Not sure how to handle on Web-UI.
